src/Tizen.NUI/src/public/Transition/TransitionBase.cs

```diff
@@ -25,19 +25,19 @@ namespace Tizen.NUI
     /// TransitionBase class is a base class for each Transitions.
     /// Each Transition child classes inherits this base class.
     /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 9 </since_tizen>
     public class TransitionBase : Disposable
     {
         private static readonly float DefaultDuration = 0.5f;
         protected internal AlphaFunction alphaFunction = new AlphaFunction(AlphaFunction.BuiltinFunctions.Default);
         protected internal TimePeriod timePeriod = new TimePeriod(DefaultDuration);
 
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
I think you should add description of APIs.
The description is shown by VS's intellisense.

Reply:
You also need to add description to AlphaFunction & TimePerio.

Reply:
Thank you.